images/scripts/centos-7.install

```diff
@@ -3,6 +3,6 @@
 set -e
 
 # remove cockpit distro packages, testing with upstream master
-rpm --erase --verbose cockpit cockpit-ws cockpit-bridge cockpit-system cockpit-dashboard
+rpm --erase --verbose cockpit cockpit-ws cockpit-bridge cockpit-system
```

Comment:
I'd keep that -- I assume you don't want to drop c-dashboard from the cockpit rhel-7.9 branch.